assets/src/js/admin/stripe-admin.js

```diff
@@ -0,0 +1,151 @@
+		donationStatus.addEventListener( 'change', ( event ) => {
+			const stripeCheckbox = document.getElementById( 'give-stripe-opt-refund' );
+
+			if ( null === stripeCheckbox ) {
+				return;
+			}
+
+			stripeCheckbox.checked = false;
+
+			// If donation status is complete, then show refund checkbox
+			if ( 'refunded' === event.target.value ) {
+				document.getElementById( 'give-stripe-opt-refund-wrap' ).style.display = 'block';
+			} else {
+				document.getElementById( 'give-stripe-opt-refund-wrap' ).style.display = 'none';
+			}
+		} );
+	}
+
+	// Toggle based on selection of stripe fonts admin settings.
+	if ( null !== stripeFonts ) {
+		stripeFonts.forEach( ( element ) => {
+			const stripeGoogleFontsWrap = document.querySelector( '.give-stripe-google-fonts-wrap' );
+			const stripeCustomFontsWrap = document.querySelector( '.give-stripe-custom-fonts-wrap' );
+
+			element.addEventListener( 'change', ( event ) => {
+				if ( 'custom_fonts' === event.target.value ) {
+					stripeGoogleFontsWrap.style.display = 'none';
+					stripeCustomFontsWrap.style.display = 'table-row';
+				} else if ( 'google_fonts' === event.target.value ) {
+					stripeGoogleFontsWrap.style.display = 'table-row';
+					stripeCustomFontsWrap.style.display = 'none';
+				}
+			} );
+		} );
+	}
+
+	const dismissConnectBanner = document.querySelector( '.give-stripe-connect-temp-dismiss' );
+	if ( null !== dismissConnectBanner ) {
+		dismissConnectBanner.addEventListener( 'click', function( e ) {
+			// Prevent reload.
+			e.preventDefault();
+
+			// Proceed with AJAX.
+			const dismissBanner = new XMLHttpRequest();
+			const formData = new FormData();
+
+			formData.append( 'action', 'give_stripe_connect_dismiss' );
+
+			dismissBanner.onreadystatechange = function() {
+				if (
+					4 === this.readyState &&
+					200 === this.status &&
+					'success' === this.responseText
+				) {
+					const connectBanner = document.querySelector( '.give-stripe-connect-message' );
+					connectBanner.remove();
+				}
+			};
+			dismissBanner.open( 'POST', ajaxurl, false );
+			dismissBanner.send( formData );
+		} );
+	}
+
+	// Bail out, if modal or enable apple/google pay option is null.
+	if ( null === modalOption ) {
+		return;
+	}
+
+	modalCheckout.forEach( function( element, index ) {
+		if ( modalOption.checked ) {
+			modalCheckout[ index ].style.display = 'table-row';
+			ccFormatSettings.style.display = 'none';
+		}
+	} );
+
+	modalOption.addEventListener( 'click', function() {
+		modalCheckout.forEach( function( element, index ) {
+			if ( modalOption.checked ) {
+				modalCheckout[ index ].style.display = 'table-row';
+				ccFormatSettings.style.display = 'none';
+			} else {
+				modalCheckout[ index ].style.display = 'none';
+				ccFormatSettings.style.display = 'table-row';
+			}
+		} );
+	} );
+} );
+
+/**
+ * This function will help to beautify JSON data.
+ *
+ * @param element
+ * @param value
+ *
+ * @since 2.5.0
+ */
+function prettyJson( element, value ) {
```

Comment:
@mehul0810 global helper function should be accessible via `Give.fn`. Let me know if you have any question for Give JS API